components/src/molecules/rate/Rate.stories.js

```diff
@@ -0,0 +1,95 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import ARate from './Rate.vue'
+
+const info = `
+  ---
+  #### Use these modifiers with \`.radio\` class.
+  - \`.rating--large\` - Selector for applying large styles
+  ---
+`
+
+storiesOf('Molecules/Rate', module)
+  .addParameters({ info })
+  // @vue/component
+  .add('Default', () => ({
+    components: { ARate },
+    data: () => ({
+      rating: 2.4,
+      ratingItems: 5
+    }),
+    template: `
+       <a-rate 
+        v-model="rating"
+        :rating-items="ratingItems"
+        disabled
+        legend="Your rating"
+       />
+    `
+  }))
+  // @vue/component
+  .add('Rate', () => ({
+    components: { ARate },
+    data: () => ({
+      rating: 1,
+      ratingItems: 4
+    }),
+    methods: {
+      select: action('Selected'),
+      ariaLabelRate () {
```

Comment:
Move to `computed`

Reply:
I moved it as a default value, imo we don't need it in computed, what do you think?